worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -99,14 +100,42 @@ public JobTrackingWorkerReporter() throws JobReportingException {
     /**
      * Reports the percentage completed of the specified task id.
      *
-     * @param jobTaskId identifies the job task whose progress is to be reported
+     * @param jobTaskId identifies the job task in progress
      * @param estimatedPercentageCompleted an indication of progress on the job task
      * @throws JobReportingException if a failure occurs in connecting or reporting to a Job Database
      */
     @Override
     public void reportJobTaskProgress(final String jobTaskId, final int estimatedPercentageCompleted) throws JobReportingException
     {
-        LOG.trace("Received progress update message for task {}; taking no action", jobTaskId);
+        final JobTaskId jobTaskIdObj = JobTaskId.fromMessageId(jobTaskId);
+        final String partitionId = jobTaskIdObj.getPartitionId();
+        final String jobId = jobTaskIdObj.getJobId();
+        final String taskId = jobTaskIdObj.getId();
+
+        LOG.trace("Received progress update message for task {} / {} .Progress: {};", taskId, partitionId, estimatedPercentageCompleted);
+
+        try (final Connection conn = getConnection()) {
+            try (final CallableStatement stmt = conn.prepareCall("{call report_progress(?,?,?,?)}")) {
+                stmt.setString(1, partitionId);
+                stmt.setString(2, jobId);
+                stmt.setString(3, taskId);
+                stmt.setDouble(4, estimatedPercentageCompleted);
+
+                executeToJobDependencyList(stmt);
```

Comment:
The `report_progress` function shouldn't return a job dependency list.

Reply:
True, done